src/libraries/System.Memory/src/System/Buffers/Text/Base64Decoder.cs

```diff
@@ -552,17 +574,17 @@ private static unsafe void Ssse3Decode(ref byte* srcBytes, ref byte* destBytes,
             // 1111 0x10 andlut     0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10 0x10
 
             // The JIT won't hoist these "constants", so help it
-            Vector128<sbyte> lutHi = Vector128.Create(
+            Vector128<byte> lutHi = Vector128.Create(
                 0x10, 0x10, 0x01, 0x02,
                 0x04, 0x08, 0x04, 0x08,
                 0x10, 0x10, 0x10, 0x10,
-                0x10, 0x10, 0x10, 0x10);
+                0x10, 0x10, 0x10, 0x10).AsByte();
 
-            Vector128<sbyte> lutLo = Vector128.Create(
+            Vector128<byte> lutLo = Vector128.Create(
```

Comment:
nit, just add a `(byte)0x10` to one of the parameters and you won't need to `AsByte`. It will result in smaller and more efficient IL.

Reply:
(Same with the other LUT above)

Reply:
If you're going to have `.AsByte()` then manually packing this to `4x uint` or `2x ulong` (e.g. `Vector128.Create(0x02011010, 0x08040804, 0x10101010, 0x10101010)`) would also result in smaller IL

Passing in 16 arguments results in 32-bytes, vs the 20 for `4x uint` and 17 for `2x ulong`.

Reply:
That's much neater too. Will do.